tabs/osd.js

```diff
@@ -762,7 +769,8 @@ OSD.chooseFields = function () {
     } else {
       OSD.constants.DISPLAY_FIELDS = OSD.constants.DISPLAY_FIELDS.concat([
         F.TIMER_1,
-        F.TIMER_2
+        F.TIMER_2,
+        F.REMAINING_TIME_ESTIMATE
```

Comment:
That's the wrong place. Has to go https://github.com/JyeSmith/betaflight-configurator/blob/7d34f9f6631fe8e64fc50f25a1651130c14947fe/tabs/osd.js#L827.